### DISCUSSION TITLE
Cant export group chat message and channel conversation

### DISCUSSION THREAD

**Discussion by stephwon** [Tyrrrz/DiscordChatExporter#624] [Q&A]
### Version

2.28

### Flavor

GUI (Graphical User Interface)

### Export format

HTML

### Details

I tried to export my group chat but the chat does not appear on extractor.
Even on channels where I have access the extractor does not fully extract (1 day worth of conversation)

### Steps to reproduce

Basically the same step

**Comment by Tyrrrz** [comment]
> I tried to export my group chat but the chat does not appear on extractor.

Did you possibly close the chat in Discord (i.e. press the X button)?

> Even on channels where I have access the extractor does not fully extract (1 day worth of conversation)

What do you mean by "fully"?

**Comment by stephwon** [comment]
For example when I set a parameter to extract 1 day worth of chat, it only extracted 6 messages instead of extracting 1 day worth of conversation.

I did not close the chat in Discord, i had the chat open
